setup.py

```diff
@@ -58,7 +58,7 @@
     # wheels, ensure that their names encode the version they are targeting. For
     # example, a wheel named 'cirq-#.#.#-py27-none-any.whl' will only be
     # installed in python 2.7.
-    python_requires=('>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*, !=3.5.0, '
+    python_requires=('!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*, !=3.5.0, '
```

Comment:
Can this be simplified?
```suggestion
    python_requires=('>=3.5.2'),
```

Reply:
Sure. Since this Pull Request is about Python 2 related stuff I have opened a new request for this: https://github.com/quantumlib/Cirq/pull/1479

Reply:
Make sure to accept this change too, it was one suggestion per line. (I don't think GitHub allows multiline suggestions.)